pkgs/apps/psi4/default.nix

```diff
@@ -16,7 +16,6 @@
 , qcelemental
 , qcengine
 , numpy
-, pylibefp
```

Comment:
If it is removed. Is the upgrade below still needed?

Reply:
Not necessarily. 0.6.1 is not the latest release, that is 0.6.2. Should I upgrade go to this then?

Reply:
Yes, a release is always cleaner than upgrading to unstable (unless it is really necessary).